src/scenes/WeatherHourlyList/WeatherHourly.js

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+/* eslint-disable camelcase */
```

Comment:
Should not disable camelCase. Write a function to convert response properties to camelCase and rename these props to camelCase.
And don't forget to add prop types

Reply:
done